features/account/registering.feature

```diff
@@ -0,0 +1,33 @@
+@customer_account
+Feature: Account registration
```

Comment:
I would put this feature in a user folder. A registering subfolder has been placed there already.